sssd_test_framework/utils/sssd.py

```diff
@@ -1081,3 +1081,9 @@ def socket_responders(self, responders: list[str] | None = None) -> None:
         for responder in responders:
             socket_unit = f"sssd-{responder}.socket"
             self.sssd.svc.start(socket_unit)
+
+    def subid(self) -> None:
+        """
+        Configure SSSD for subid.
+        """
+        self.sssd.host.conn.run('echo "subid:      sss" >> /etc/authselect/nsswitch.conf')
```

Comment:
I tried 

```
self.sssd.fs.append(
    "/etc/authselect/nsswitch.conf",
    "subid:      sss\r\n\r\n"
)
```

For some reason, SSSD would not process the file until it was manually opened and closed. I tried several combinations; this is probably a bug with pytest-mh

Reply:
I don't think you need to edit '/etc/authselect/nsswitch.conf' manually.

Rather use `authselect enable-feature with-subid`

Reply:
You are absolutely right, updated.

Reply:
I see using `authselect.select("sssd", ["with-subid"])` is a pattern:
```
utils/authselect.py:21:            client.authselect.select('sssd', ['with-mkhomedir'])
utils/sshd.py:30:            client.authselect.select("sssd", ["with-gssapi", "with-mkhomedir"])
utils/smartcard.py:185:        client.authselect.select("sssd", ["with-smartcard"])
utils/sssd.py:919:        self.sssd.authselect.select("sssd", ["with-sudo"])
utils/sssd.py:930:        self.sssd.authselect.select("sssd", ["with-gssapi", "with-sudo"])
utils/sssd.py:943:        self.sssd.authselect.select("sssd")
utils/sssd.py:953:        self.sssd.authselect.select("sssd", ["with-mkhomedir"])
utils/sssd.py:1044:        self.sssd.authselect.select("sssd", features)
```
but why not `enable-feature with-subid`?